run_unittests.py

```diff
@@ -5607,6 +5607,24 @@ def test_identity_cross(self):
         # TODO should someday be explicit about build platform only here
         self.init(testdir, override_envvars=env)
 
+    def test_static_link(self):
+        if 'clang' in os.environ.get('CC', ''):
+            raise unittest.SkipTest("Clang doesn't support LIBRARY_PATH.")
+
+        # Build some libraries and install them
+        testdir = os.path.join(self.unit_test_dir, '69 static link/lib')
+        libdir = os.path.join(self.installdir, self.prefix[1:], self.libdir)
+        self.init(testdir)
+        self.install()
+
+        # Test that installed libraries works
+        self.new_builddir()
+        testdir = os.path.join(self.unit_test_dir, '69 static link')
+        env = {'LIBRARY_PATH': libdir}
```

Comment:
Do not use envvars whenever possible. Use the link args command line argument instead.

Reply:
Ok, replaced LIBRARY_PATH by `'-Dc_link_args="-L{}"'.format(libdir)`, but I don't think there is an alternative to PKG_CONFIG_LIBDIR and LD_LIBRARY_PATH, is it?